### PR TITLE
#160452442: Like and dislike functionality

### PR DESCRIPTION
**Pull request by uchemukolo**
#### What does this PR do?
Handle like and dislike functionality on an article
#### Description of Task to be completed?
- Write tests for actions, reducer
- Create an action to handle like and dislike on an article
- Add API route to like and dislike an article
#### How should this be manually tested?
- Start the application using `npm start`
- Navigate to `localhost:3000`.
- Signup/login 
- Click on any article 
- When article displays, click on the like or dislike icon

#### What are the relevant pivotal tracker stories?
- [#160452442](<https://www.pivotaltracker.com/story/show/160452442>)
#### Questions:

### CODE REVIEW

File: src/tests/reducers/likeReducer.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import * as types from '../../redux/constants/articles';
+import likeReducer from '../../redux/reducer/article/likes';
+
+const initialState = {
+  articleLiked: false,
+  articleDisliked: false,
+};
+describe.only('Like Reducer', () => {
+  // it('it should like an article', (done) => {
```
Comment (blackshady) [inline src/tests/reducers/likeReducer.test.js]: Nice job please remove the commented lines of code if you no longer need them.

Comment (uchemukolo) [reply src/tests/reducers/likeReducer.test.js]: Okay. I will remove them and update the PR

Comment (blackshady) [reply src/tests/reducers/likeReducer.test.js]: Great!

File: src/tests/reducers/likeReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  articleDisliked: false,
+};
+
+const reaction = {
+  status: 'like',
+  likeCount: 0,
+  dislikeCount: 0
+};
+
+describe.only('Like Reducer', () => {
```
Comment (blackshady) [inline src/tests/reducers/likeReducer.test.js]: Nice job on increasing your test coverage. Please why do you still have `only` on the describe block.

Comment (uchemukolo) [reply src/tests/reducers/likeReducer.test.js]: @blackshady The `only` has been removed. It was an oversight.

Comment (blackshady) [reply src/tests/reducers/likeReducer.test.js]: Okay great!

**Review by blackshady — APPROVED** [review]
I have reviewed and tested this PR. It meets the expected requirements stated in the PT board [#160452442](https://www.pivotaltracker.com/story/show/160452442)
It looks good to me.

**Pull request merged**